src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -274,6 +283,50 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
             </GridArea>
             <OakGridArea $colSpan={[12, 9]}>
               <OakFlex $flexDirection={"column"} $position={"relative"}>
+                {pageLinks.find((p) => p.label === "Lesson guide") &&
+                  lessonGuideUrl && (
+                    <LessonItemContainer
+                      isSpecialist={isSpecialist}
+                      ref={lessonGuideSectionRef}
+                      title={"Lesson guide"}
+                      //Defaulted to false until download ticket implementation
+                      downloadable={false}
+                      // Avo types need to be updated to include lesson guide
```

Comment:
do we have a ticket for this?

Reply:
Yeah working on it atm https://www.notion.so/oaknationalacademy/12926cc4e1b18015a518efe15426e53b?v=ab3cc64052624acaa90f016720b46001&p=12f26cc4e1b180b9954bfaf991d2416f&pm=s